lib/entry.js

```diff
@@ -98,21 +100,39 @@ function SalesforceEntity(callScope) {
     });
     conn.on('refresh', (accessToken, res) => {
       console.log('Keys were updated, res=%j', res);
-      emitKeys({oauth: res});
+      emitKeys({ oauth: res });
     });
 
-    await conn.sobject(cfg.object)
+    const maxFetch = cfg.sizeOfPollingPage || 1000;
+    const singlePagePerInterval = (cfg.singlePagePerInterval !== 'no') ;
+    const maxTime = cfg.endTime ? ` AND LastModifiedDate < ${cfg.endTime}` : '';
+    let hasMorePages = true;
+    let lastSeenTime = snapshot.previousLastModified;
+
+    do {
+      const whereCondition = `LastModifiedDate > ${lastSeenTime}${maxTime}`;
```

Comment:
`>` could be `>=` in some cases based on our last discussion on OIH standards.

Reply:
done (rebase commit from correct branch)